Websocket path wrong prefix in ws/status

### Description

Dear developers,

it seems that the websocket_urlpatterns in urls.py use the wrong prefix. It seems that the channels middleware automatically constructs an url relative to the BASE_URL.

So consider the following setup with paperless-ngx hosted at a subpath with 

```
PAPERLESS_FORCE_SCRIPT_NAME=/paperless
https://example.com/paperless
BASE_URL=/paperless/ (as constructed by _parse_base_paths())
```

With that config a request to https://example.com/paperless/ws/status returns a 403. Requesting https://example.com/paperless/paperless/ws/status returns the desired websocket status endpoint.

Removing the BASE_URL from the websocket_urlpatterns results in a request to https://example.com/ws/status to return successfully with 101 as expected.

```
diff --git a/src/paperless/urls.py b/src/paperless/urls.py
index 39819fd3d..007d83355 100644
--- a/src/paperless/urls.py
+++ b/src/paperless/urls.py
@@ -363,7 +363,7 @@ urlpatterns = [
 
 
 websocket_urlpatterns = [
-    path(settings.BASE_URL.lstrip("/") + "ws/status/", StatusConsumer.as_asgi()),
+    path("ws/status/", StatusConsumer.as_asgi()),
 ]
 
 # Text in each page's <h1> (and above login form).

```

This seems to be due to a recent change in the channels upstream, which made channels BASE_URL-aware. I have been running paperless-ngx for years and clearly remember that I had to rewrite the websocket urls manually in the proxy to work around that channels bug.

Then commit #2371 added a workaround for the upstream channels bug in paperless-ngx. Now that upstream fixed the original bug, the now unnecessary paperless-ngx  workaround adds an additional path component, breaking the websocket in deployments in a subpath.

Im going to prepare a PR and would kindly ask for merge. I am aware of the numerous bug reports related to that issue but this is not an upstream bug in channels, just a workaround that is no longer required on the paperless-ngx side.


### Steps to reproduce

1. Deploy paperless-ngx on a subpath
2. Open the web frontend in a browser and observe an 403 error when accessing [...]/subpath/ws/status/

### Webserver logs

```bash
2025-06-16 15:52:40,825] [ERROR] [_granian.utils] Application callable raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/site-packages/granian/_futures.py", line 15, in future_watcher
    await inner(watcher.scope, watcher.proto)
  File "/usr/local/lib/python3.12/site-packages/channels/routing.py", line 48, in __call__
    return await application(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/channels/sessions.py", line 44, in __call__
    return await self.inner(dict(scope, cookies=cookies), receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/channels/sessions.py", line 261, in __call__
    return await self.inner(wrapper.scope, receive, wrapper.send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/channels/auth.py", line 185, in __call__
    return await super().__call__(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/channels/middleware.py", line 24, in __call__
    return await self.inner(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/channels/routing.py", line 136, in __call__
    raise ValueError("No route found for path %r." % path)
ValueError: No route found for path 'ws/status'.
```

### Browser logs

```bash

```

### Paperless-ngx version

2.16.3

### Host OS

Linux

### Installation method

Docker - official image

### System status

```json

```

### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

--- a/src/paperless/urls.py
+++ b/src/paperless/urls.py
@@ -363,7 +363,7 @@ urlpatterns = [
 
 
 websocket_urlpatterns = [
-    path(settings.BASE_URL.lstrip("/") + "ws/status/", StatusConsumer.as_asgi()),
+    path("ws/status/", StatusConsumer.as_asgi()),
 ]
 
 # Text in each page's <h1> (and above login form).

